Sources/Cirrus/SyncEngine.swift

```diff
@@ -98,7 +98,7 @@ public final class SyncEngine<Model: CloudKitCodable> {
 
     self.logHandler =
       logHandler ?? { string, level in
-        if #available(iOS 14.0, macOS 11.0, *) {
+        if #available(iOS 14.0, macOS 11.0, watchOS 7.0, tvOS 14.0, *) {
```

Comment:
Mind updating this to match the Package.swift? I think we can do `iOS 14.0, macOS 11.0, watchOS 6.0, tvOS 13.0` in both places

Reply:
Hi @jayhickey  
This was intentional. Logger is only available from `watchOS 7.0` and `tvOS 14.0` respectively. ([reference](https://developer.apple.com/documentation/os/logger))